agent/src/softwarecontaineragent.cpp

```diff
@@ -42,7 +42,7 @@ SoftwareContainerAgent::SoftwareContainerAgent(
         m_shutdownContainers = !config.getBoolValue(ConfigDefinition::SC_GROUP,
                                                     ConfigDefinition::SC_KEEP_CONTAINERS_ALIVE_KEY);
     } catch (ConfigError &error) {
-        throw ReturnCode::FAILURE;
+        throw;
```

Comment:
You don't have to catch if you're just going to throw the same without doing anything else.